### PR TITLE
Added the '--new-comment ' argument

### PR DESCRIPTION
**Pull request by bruno-rocha-movile**
Implementing [#618](https://github.com/danger/danger/issues/618) - Currently, users that would like to have Danger post new comments on every build (instead of editing it's first one) need to generate new danger_ids on every build. This pull request wraps this process into a new argument.

I decided to use a formatted time string as the ID for visual purposes, since GitHub shows the instance's danger_id on the PR's status grid.

An obvious upgrade for this would be making the default danger_id ("danger") update according to the usage of this argument, so that the following process:
```
bundle exec danger
bundle exec danger --new-comment
bundle exec danger
```
..would make Danger update the comment created at the second call, instead of the first one. But since most CI servers wipe their workspaces before the builds, I could not find a decent way to implement this. This could also be used to remove the previous build's check results, if the user would like only one check to be displayed. If you guys have any suggestion for this, I would be happy to add this functionality to the PR.

### CODE REVIEW

File: lib/danger/commands/runner.rb [hunk 1]
```diff
@@ -56,7 +57,8 @@ def self.options
         ["--head=[master|dev|stable]", "A branch/tag/commit to use as the head"],
         ["--fail-on-errors=<true|false>", "Should always fail the build process, defaults to false"],
         ["--dangerfile=<path/to/dangerfile>", "The location of your Dangerfile"],
-        ["--danger_id=<id>", "The identifier of this Danger instance"]
+        ["--danger_id=<id>", "The identifier of this Danger instance"],
+        ["--new-comment", "Makes the build process ignore other Danger instances, resulting in a new comment. Overrides '--danger_id'."]
```
Comment (JuanitoFatas) [inline lib/danger/commands/runner.rb]: IMHO a little bit too implementation specific 😅? How about something like `Makes Danger post a new comment for each build`😄?

**Comment by JuanitoFatas** [comment]
Would you show some screenshots for this danger new comment functionality? Thanks!

File: lib/danger/commands/runner.rb (hunk 1 above)
Comment (bruno-rocha-movile) [inline lib/danger/commands/runner.rb]: I didn't wanted to give the impression that removing the argument on a subsequent build would make Danger edit the new comment... I think someone could be confused by it, what do you think?

**Comment by bruno-rocha-movile** [comment]
Here's what happens:
screen shot 2016-10-10 at 2 41 05 pm

It makes it easier to read your pull request, depending on your team's rules for pull requests. In my team's case, it's wonderful :)

**Comment by orta** [comment]
Hah - that avatar

**Comment by JuanitoFatas** [comment]
Would you like to join the org? @bruno-rocha-movile /cc @orta

File: lib/danger/commands/runner.rb (hunk 1 above)
Comment (JuanitoFatas) [inline lib/danger/commands/runner.rb]: 👍

**Comment by orta** [comment]
👍 and @JuanitoFatas I've given you root org access to @danger now too - you've been doing an awesome job

**Comment by bruno-rocha-movile** [comment]
@JuanitoFatas @orta of course! Thanks for the invite! I have some other interesting improvements that I was planning to do as well, Danger is amazing :)

**Comment by orta** [comment]
Sent!

**Pull request merged**